### ISSUE TITLE
About Robust-LGB

### ISSUE THREAD

**Issue by wishforgood** [Luojiaqimath/Robust-GBDT#1]
Hi, I'm new to this area and interested in your work. Could you please show me how to apply your loss to LightGBM, so as to refine it in robustness?

**Comment by wishforgood** [comment]
Specifically, how to determine the value of r and q?

**Comment by Luojiaqimath** [comment]
To apply the loss to LightGBM (using the sklearn interface), you need to set the parameter 'objective' to either 'RFLBinary' or 'LGBRFLMulti'.

The optimal values for r and q can vary depending on your dataset and task. It's a good idea to experiment with different values and observe how they affect the model's performance on a validation set.

**Comment by wishforgood** [comment]
Ok, thanks for your help, I will try it later
